server/types/serialization.go

```diff
@@ -26,6 +27,14 @@ const (
 	serializationID_Bool      serializationID = 1
```

Comment:
Why aren't these iota + 1

Reply:
If the goal is to use hand-managed serialization ids, we should probably go with the postgres OIDs instead